### PR TITLE
bugfix: dont remove `\`

### PR DESCRIPTION
**Pull request by densumesh**

### CODE REVIEW

File: server/src/operators/organization_operator.rs [hunk 1]
```diff
@@ -629,7 +629,7 @@ pub async fn update_all_org_dataset_configs_query(
 ) -> Result<(), ServiceError> {
     let concat_configs_raw_query = sql_query(format!(
         "UPDATE datasets SET server_configuration = server_configuration || '{}' WHERE organization_id = '{}';",
-        new_config.to_string().replace('\'', "''").replace('\\', ""), org_id
+        new_config.to_string().replace('\'', "''"), org_id
```
Comment (skeptrunedev) [inline server/src/operators/organization_operator.rs:632]:
```suggestion
        new_config.to_string().replace('\'', "''"), org_id
```

Comment (skeptrunedev) [reply server/src/operators/organization_operator.rs:632]: Shouldn't there only be one single quote in the second param spot?

**Review by skeptrunedev — CHANGES_REQUESTED** [review]

File: server/src/operators/organization_operator.rs (hunk 1 above)
Comment (densumesh) [reply server/src/operators/organization_operator.rs:632]: No to escape the ' in Postgres you double quote

Comment (densumesh) [reply server/src/operators/organization_operator.rs:632]: https://stackoverflow.com/questions/12316953/insert-text-with-single-quotes-in-postgresql

Comment (skeptrunedev) [reply server/src/operators/organization_operator.rs:632]: Damn, alright

**Review by skeptrunedev — APPROVED** [review]

**Pull request merged**